File: Java/Exercism/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        
        double powerSum = Math.hypot(xOfDart, yOfDart);
     
         if (powerSum <= 1){
            return 10;  
         }
         if (powerSum <= 5){
            return 5;  
         }
         if (powerSum <= 10){
            return 1;  
         }
         if (powerSum > 10){
            return 0;  
         }
    return 0;
    }
}

File: JavaScript/Exercism/Darts.js
//
// This is only a SKELETON file for the 'Darts' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const score = (x, y) => {
  let result = Math.hypot(x, y);

  if (result <= 1){
    return 10;
  
  } else if (result <= 5){
    return 5; 
  
  } else if (result <= 10){
    return 1;
  
  } else {
    return 0;
    
  }
};